src/components/activity/ActivitySessionItem.tsx

```diff
@@ -181,7 +181,7 @@ const ActivitySessionItem: React.VFC<{
           {session.threshold && (
             <span>
               {formatMessage(productMessages.activity.content.least)}
-              {session.threshold}
+              {isNaN(Number(session.threshold)) ? 'N/A' : Math.round(parseFloat(session.threshold))}
```

Comment:
The situation here is simply to get an integer.
could be simply using `Math.floor()`

Reply:
it is better to convert to use `toFixed`
```
isNaN(Number(session.threshold)) ? 'N/A' : Number(session.threshold).toFixed(0)
```

it is much better to convert into number earlier